### ISSUE TITLE
Stack feed filter

### ISSUE THREAD

**Issue by bsmith11** [bsmith11/Stack#17]
Add ability to filter the sources that are displayed in the Stack feed

**Comment by sheady** [comment]
@bsmith11 by filtering are you wanting the ability to remove just 1(or more) source(s)? eg see everyone except sludgebrown?

**Comment by bsmith11** [comment]
Yes. This would only apply in the "Stack" feed. We need to find a good place to display the filters, whether it be in the Sources screen or in Settings
